new-nextjs-client/src/components/IndexPage/AddYearDropdown/AddYearDropdown.module.scss

```diff
@@ -1,3 +1,3 @@
 .container {
-  width: 12.7rem;
+  width: 11.5rem;
```

Comment:
Is AddYearDropDown component stays in the same style for small screen size?
I remember that this style makes drop-down style changes if the width is not wide enough, so I just wanted to make sure.